models/LoginForm.php

```diff
@@ -93,22 +85,38 @@ function ($attribute) {
             ],
             'rememberMe' => ['rememberMe', 'boolean'],
         ];
+
+        if(!Yii::$app->getModule('user')->debug)
```

Comment:
Code needs to follow PSR-2 codestyle. So we need space after if and brackets around if body.